Assignment submission 1

taono opened the pull request:
- added an index page.php
- test pull request.

Commit:
Assignment submission 1

* added an index page.php

Commit:
Added style.css

* this is a pull request test

zoe-1 commented:
How is the assignment coming? almost there?

Commit:
First Rough Draft

* I can't move around 'parabox' even though I tried various positioning methods.
* I will make a new login form. The current form is just a simple draft.

taono commented:
just submitted a rough draft.

zoelogic commented:
@taono great job submitting your first PR all by your self!  
Good work :-)

Commit:
Changes on style.css

* changed parabox z index to 20 and tested top/let top/right combination. Parabox still doesn't move
* positon of linkbox affects the postion of parabox.

taono commented:
I changed linkbox position and z-index of parabox. Still, parabox is not moving. Can you help to check again what went wrong? I will add hanf_form in my next commit. Thanks :-)

zoelogic commented:
@taono Your comments are PHP style comments.
CSS does not allow "//" style comments, so the above "// the box background color ...." breaks this style.
See my remarks on ./style.css

Plus, try testing what happens when you use:
- position: fixed; 
- position: absolute;
- and others..

Commit:
Thrid Draft

* added form and handle form. The login page doesn't display name and email properly. I will double check.
* treid to add rebox to the homepage. it doesn't work well.

zoe-1 commented:
@taono commit your changes to git and push them to github. 
That way I can see the current state of your project and look at your redbox.
Thx

taono commented:
yes I did. It's called the third draft php style comment tread.

Commit:
added bash assignment

* added bash answers

Commit:
Modified handle form

* modified handle form.
* new handle form in studyphp doesn't work. need to fix it.

Commit:
Fixed hello handle form

* syntax error: forget to end php variable with ;
* comemnts in php could be //  but in html should be <!-- somehting-->

Commit:
Post Assingment 1 edit

* fixed redbox issue: _forget to close one style with }._
* changed bash_guide into md file. _is there a way to visualize md codes outisde of git hub?_

Commit:
Update bash_guide.md

new line enables ##
there needs to be a space after - or * if intends to start a sentence with bullet points.

Commit:
Add phpinfo.php file

* failed to checkout to inno-code/simplephp branch.

Commit:
Deleted bash_guide.txt

* how to track modified content?

Commit:
Merge branch 'master' of https://github.com/taono/simplephp
* attempt to solve the problem of push rejection

Commit:
Deleted ./experiment folder and created ./guides

* merge required on inno-code master?
